src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesFolderProperty/config.jelly

```diff
@@ -0,0 +1,8 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <f:section title="Kubernetes Plugin">
+  	<f:entry title="${%Allowed Clouds}" field="allowedClouds">
```

Comment:
This could be more structured, like providing a list of checkboxes (one line per cloud instance) and storing a collection of cloud names in the folder property class.